dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/hive/HiveMetadataConnectorTest.java

```diff
@@ -76,14 +76,43 @@ public void testConnector() throws Exception {
     testConnectorDefaults(connector);
   }
 
-  @DataPoints("expectedTaskNames")
+  @DataPoints("commonTaskNames")
   public static final ImmutableList<String> EXPECTED_TASK_NAMES =
-      ImmutableList.of("catalogs.jsonl", "databases.jsonl");
+      ImmutableList.of(
+          "compilerworks-metadata.yaml",
+          "compilerworks-format.txt",
+          "schemata.csv",
+          "tables.jsonl",
+          "functions.csv");
```

Comment:
Do these names represent all tasks that are added by the Hive connector? Or is there a task which is added by the connector, but doesn't have its name listed here?

Reply:
The list of all the tasks depends on the value of the flag `-Dhiveql.migration.metadata=`. This list represents the names of the tasks that are always extracted no matter what the value of the flag is. The second list (i.e. `migrationMetadataTaskNames`) has all the tasks with the flag enabled.